integration-tests/ui/src/test/resources/migration/MIGR_TOOLS/xsk-test-app/xsk-test-app-copy.hdi

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why the filename is `xsk-test-app-copy.hdi`, shouldn't it be `xsk-test-app.hdi`?